actors/common/src/main/java/org/sunbird/learner/actors/otp/service/OTPService.java

```diff
@@ -26,26 +33,26 @@ public void deleteOtp(String type,String key){
     otpDao.deleteOtp(type,key);
   }
 
-  public static String getOTPSMSBody(Map<String, String> smsTemplate) {
-    try {
-      Properties props = new Properties();
-      props.put("resource.loader", "class");
-      props.put(
-          "class.resource.loader.class",
-          "org.apache.velocity.runtime.resource.loader.ClasspathResourceLoader");
-
-      VelocityEngine ve = new VelocityEngine();
-      ve.init(props);
-      Template t = ve.getTemplate("/OTPSMSTemplate.vm");
-      VelocityContext context = new VelocityContext(smsTemplate);
-      StringWriter writer = new StringWriter();
-      t.merge(context, writer);
-      return writer.toString();
-    } catch (Exception ex) {
-      ProjectLogger.log(
-          "OTPService:getOTPSMSBody: Exception occurred with error message = " + ex.getMessage(), ex);
-    }
-    return "";
+  public static String getSmsBody(String templateFile, Map<String, String> smsTemplate) {
+   try {
+    Properties props = new Properties();
+    props.put("resource.loader", "class");
+    props.put(
+            "class.resource.loader.class",
+            "org.apache.velocity.runtime.resource.loader.ClasspathResourceLoader");
+
+    VelocityEngine ve = new VelocityEngine();
+    ve.init(props);
+    Template t = ve.getTemplate("/"+templateFile);

```

Comment:
Read template from DB only , no need to put inside file.

Reply:
Now sms templates are reading from DB.

Reply:
is cql taken care for other env as well.